sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp

```diff
@@ -2738,8 +2738,8 @@ namespace Azure { namespace Storage { namespace Blobs {
             Storage::Details::XmlWriter& writer,
             const SetServicePropertiesOptions& options)
         {
-          writer.Write(Storage::Details::XmlNode{
-              Storage::Details::XmlNodeType::StartTag, "StorageServiceProperties"});
+          writer.Write(Storage::Details::XmlNode{Storage::Details::XmlNodeType::StartTag,
+                                                 "StorageServiceProperties"});
           BlobServicePropertiesToXml(writer, options.Properties);
```

Comment:
I used clang-format 11 to format this code. How come we got different result?

Reply:
Was that on Windows?  
I can try to reproduce. Please share your env details and where did you install clang-format from.

If there's a difference from clang-format 9 vs 11, we will need to figure it out

Reply:
I feel there's something wrong with my clang-format tool. please give me some time to investigate.

Reply:
This is what I got from clang-format 9
![image](https://user-images.githubusercontent.com/52266153/104310018-02371700-550e-11eb-9028-0f51eb357bc9.png)

This is what I got from clang-format 11

![image](https://user-images.githubusercontent.com/52266153/104310096-1b3fc800-550e-11eb-8b74-91003a35a5e9.png)


The result is the same as 11.0 for the built-in one of Visual Studio.

Reply:
I think we should use the latest version or the built-in one of Visual Studio

Reply:
I have created PRs for clang format 9, 10 and 11:
#1333, #1334 , #1335

We need to review and decide which one we want